src/layouts/Media/Media.tsx

```diff
@@ -0,0 +1,184 @@
+import { SimpleGrid, Box, Text, Skeleton } from "@chakra-ui/react"
+import { Button } from "@opengovsg/design-system-react"
+import _ from "lodash"
+import { BiBulb, BiUpload } from "react-icons/bi"
+import { Link, Switch, useRouteMatch, useHistory } from "react-router-dom"
+
+import { useGetMediaFolders } from "hooks/directoryHooks"
+
+import { DeleteWarningScreen } from "layouts/screens/DeleteWarningScreen"
+import { DirectoryCreationScreen } from "layouts/screens/DirectoryCreationScreen"
+import { DirectorySettingsScreen } from "layouts/screens/DirectorySettingsScreen"
+import { MediaCreationScreen } from "layouts/screens/MediaCreationScreen"
+import { MediaSettingsScreen } from "layouts/screens/MediaSettingsScreen"
+import { MoveScreen } from "layouts/screens/MoveScreen"
+
+import { ProtectedRouteWithProps } from "routing/ProtectedRouteWithProps"
+
+import { isDirData, isMediaData } from "types/utils"
+
+import {
+  CreateButton,
+  Section,
+  SectionCaption,
+  SectionHeader,
+} from "../components"
+import { SiteViewLayout } from "../layouts"
+
+import {
+  MediaDirectoryCard,
+  ImagePreviewCard,
+  FilePreviewCard,
+  MediaBreadcrumbs,
+} from "./components"
+
+interface MediaLabels {
+  singularMediaLabel: "file" | "image"
+  pluralMediaLabel: "files" | "images"
+  directoryLabel: "directory" | "album"
+}
+
+// Utility method to help ease over the various labels associated
+// with the media type so that we can avoid repeated conditionals
+const getMediaLabels = (mediaType: "files" | "images"): MediaLabels => {
+  if (mediaType === "files") {
+    return {
+      singularMediaLabel: "file",
+      pluralMediaLabel: "files",
+      directoryLabel: "directory",
+    }
+  }
+
+  return {
+    singularMediaLabel: "image",
+    pluralMediaLabel: "images",
+    directoryLabel: "album",
+  }
+}
+
+export const Media = (): JSX.Element => {
+  const history = useHistory()
+  const { params, path, url } = useRouteMatch<{
+    siteName: string
+    mediaRoom: "files" | "images"
+    mediaDirectoryName: string
+  }>()
+  const { mediaRoom: mediaType } = params
+  const { data: mediasData, isLoading } = useGetMediaFolders(params)
+  const {
+    singularMediaLabel,
+    pluralMediaLabel,
+    directoryLabel,
+  } = getMediaLabels(mediaType)
+
+  return (
+    <>
+      <SiteViewLayout overflow="hidden">
+        <Section>
+          <Box>
+            <Text as="h2" textStyle="h2">
+              {_.upperFirst(mediaType)}
+            </Text>
+            <MediaBreadcrumbs />
+          </Box>
+        </Section>
+        <Section>
+          <SectionHeader label="Albums">
+            <CreateButton as={Link} to={`${url}/createMedia`}>
+              {`Create ${directoryLabel}`}
+            </CreateButton>
+          </SectionHeader>
```

Comment:
This link should be to `${url}/createDirectory`!

Reply:
good catch, thanks!